auth-web/src/components/auth/CreateUserProfileForm.vue

```diff
@@ -191,7 +191,7 @@ export default class CreateUserProfileForm extends Mixins(NextPageMixin) {
     }
 
     private cancel () {
-      window.history.back()
+      this.$router.push('/signin/bcros/')
```

Comment:
where does it navigate if the user is signed in?

Reply:
we can navigate to the home page if any other user is already logged in, also will that case happen in the real users?